images/s6_assets/init/pulpcore-api

```diff
@@ -13,8 +13,28 @@ else
   PULP_API_ENTRYPOINT=("gunicorn" "pulpcore.app.wsgi:application" "--bind" "127.0.0.1:24817" "--name" "pulp-api" "--access-logformat" "pulp [%({correlation-id}o)s]: %(h)s %(l)s %(u)s %(t)s \"%(r)s\" %(s)s %(b)s \"%(f)s\" \"%(a)s\"")
 fi
 
+# Process recycling only supported for pulpcore >= 3.41.0
+# due to https://github.com/pulp/pulpcore/issues/4679
+if python3 -c \
+  'import importlib.metadata; \
+   import semantic_version; \
+   exit(\
+    int(\
+      not(\
+        semantic_version.Version(\
```

Comment:
No need to wrap this into semantic_version. Python version compare should do the trick.
Is it `from packaging.version import Version`?

Reply:
Thanks.

Changed to use `packaging.version` instead of `semantic_version`